models/response/transaction.go

```diff
@@ -0,0 +1,39 @@
+package response
+
+import "github.com/bitcoin-sv/spv-wallet/models/common"
+
+// Transaction is a model that represents a transaction.
+type Transaction struct {
+	// Model is a common model that contains common fields for all models.
+	common.Model
+	// ID is a transaction id.
+	ID string `json:"id" example:"01d0d0067652f684c6acb3683763f353fce55f6496521c7d99e71e1d27e53f5c"`
+	// Hex is a transaction hex.
+	Hex string `json:"hex" example:"0100000002..."`
+	// XpubInIDs is a slice of xpub input ids.
+	XpubInIDs []string `json:"xpub_in_ids" example:"bb8593f85ef8056a77026ad415f02128f3768906de53e9e8bf8749fe2d66cf50"`
```

Comment:
```suggestion
	XpubInIDs []string `json:"xpub_in_ids" example:["bb8593f85ef8056a77026ad415f02128f3768906de53e9e8bf8749fe2d66cf50"]`
```
I think this will be better

Reply:
applied suggestion